.idea/.gitignore

```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```

Comment:
Plsease remove all these files from the PR and add `.idea` folder to the `.gitignore` file